gammapy/utils/pbar.py

```diff
@@ -0,0 +1,36 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Utilities for progress bar display"""
+import logging
+from collections.abc import Iterable
+
+log = logging.getLogger(__name__)
+
+try:
+    from tqdm.auto import tqdm
+except ImportError:
+    class tqdm():
+        def __init__(self, iterable, disable=True, **kwargs):
+            self.disable = disable
+            self._iterable = iterable
+            if self.disable == False:
```

Comment:
Comparison of bools is directly done with `if self.disable:` or `if not self.disable`.